src/k8s-engine/wsl.ts

```diff
@@ -815,6 +819,30 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     return stdout.trim();
   }
 
+  protected async launchAgent() {
+    try {
+      this.agentprocess?.kill('SIGTERM');
+    } catch (ex) { }
+    const agentargs = ['--distribution', INSTANCE_NAME, '--exec', '/usr/local/bin/rancher-desktop-guestagent'];
+    const agentoptions: childProcess.SpawnOptions = {
+      env:         { ...process.env },
```

Comment:
Since we're not changing `env` here, we probably don't need to specify it? (It should pick up `process.env` by default, IIRC.)

Reply:
updated to remove that.